index.html

```diff
@@ -0,0 +1,264 @@
+                                    fill="#505F79" />
+                            </svg>
+                        </a></li>
+                </ul>
+                <button class="about-btn" type="submit">Get my resume</button>
+            </div>
+            <!-- My Skills -->
+            <div class="skills">
+                <ul>
+                    <li class="p">Languages
+                        <img src="img/Union.svg" alt="counter" class="counter">
+                        <ul class="lang">
+                            <li class="skill"><img src="img/javascript.svg" alt="">JavaScript </li>
+                            <li class="skill"><img src="img/html.svg" alt="">HTML</li>
+                            <li class="skill"><img src="img/css.svg" alt="">CSS</li>
+                        </ul>
+                    </li>
+                    <li class="p">Frameworks <img src="img/Union-right.svg" alt="counter" class="counter"></li>
+                    <li class="p hr">Skills <img src="img/Union-right.svg" alt="counter" class="counter"></li>
```

Comment:
As requested by the previous reviewers, kindly adjust the languages, frameworks and skill section to match the design template. Please, keep to the margins.

Your design:
![image](https://user-images.githubusercontent.com/20554583/206496310-9f554f8c-6030-416b-84ea-bcd9f4d97dc3.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/206496120-569abc39-fafb-48c9-9747-aac66516a8ea.png)